resources/components/Layout/AppLayout.tsx

```diff
@@ -1,7 +1,70 @@
 import React from 'react'
+import Grid from '@mui/material/Grid'
+import SVG from 'react-inlinesvg'
+import {Avatar, Link} from '@mui/material'
```

Comment:
you can import `Grid` together here i think

Reply:
Thank you for pointing that out. 👍